dockstore-integration-testing/src/test/java/io/dockstore/webservice/NotebookIT.java

```diff
@@ -162,6 +165,35 @@ void testRegisterCorruptNotebook() {
         assertFalse(notebook.getWorkflowVersions().get(0).isValid());
     }
 
+    @Test
+    void testUserNotebooks() {
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false, testingPostgres);
+        ApiClient apiClient = getOpenAPIWebClient(BasicIT.USER_2_USERNAME, testingPostgres);
+        WorkflowsApi workflowsApi = new WorkflowsApi(apiClient);
+        workflowsApi.handleGitHubRelease("refs/heads/simple", installationId, simpleRepo, BasicIT.USER_2_USERNAME);
```

Comment:
watch out, this is testing off a branch rather than a tag. 
I wouldn't consider this a hard and fast rule, but note that it can be easy to break accidentally if additional commits are added